### PR TITLE
Add existing VPC support

### PR DESCRIPTION
**Pull request by mirthy**
Add existing VPC support through:
- A stub vpc-existing module
- A set of variables for the stub module located in vpc-existing.tfvars that merges it into terraform.tfvars
- [An override](https://www.terraform.io/docs/configuration/override.html) of the root modules.tf file

### CODE REVIEW

File: scripts/init-variables [hunk 1]
```diff
@@ -55,5 +55,5 @@ pod-ip-range = "${POD_IP_RANGE}"
 s3-bucket = "${AWS_ACCOUNT_ID}-${CLUSTER_NAME}-${AWS_REGION}"
 service-ip-range = "${K8S_SERVICE_IP_RANGE}"
 EOF
-
+if [ -f ./vpc-existing.tfvars ]; then cat ./vpc-existing.tfvars >>terraform.tfvars ; fi
```
Comment (mirthy) [inline scripts/init-variables:66]: Not sure if there is a better way to add these to terraform.tfvars

**Comment by mirthy** [comment]
@wellsie What did you think about this PR?

**Comment by wellsie** [comment]
@mirthy - looks good. we just need to update the maps to use terraform 0.7.x syntax in the `.tfvar` file and '[]' for the maps in the `.tf` files. i'll take a swing at that over the next few days

**Comment by mirthy** [comment]
@wellsie Took a crack at fixing the maps to terraform 0.7.x syntax

Still need to do a little more fixing, seems like the new map syntax doesn't like partial updates.

**Comment by wellsie** [comment]
👍  ty

**Pull request merged**